app/models/incident.rb

```diff
@@ -104,43 +104,53 @@ def csv_row
-          doc.state               'MA'
+          doc.state               report.state
           doc.zip                 report.zip
           doc.longitude           longitude
           doc.latitude            latitude
           doc.company             driver.division.claims_id
-          doc.incident_desc       supervisor_incident_report.try(:description)
-          doc.employee_id         driver.badge_number
-          doc.driver              driver.badge_number
           doc.driver_desc         report.description
-          doc.vehicle_route_num   report.route
+          doc.employee_entering   current_user.badge_number
+          doc.driver              driver.badge_number
           doc.vehicle_num         report.bus
-          doc.vehicle_damage_area report.damage_to_other_vehicle_point_of_impact
+          doc.speed               report.speed
+          doc.incident_desc       supervisor_incident_report.try(:description)
           doc.point_of_contact    report.damage_to_bus_point_of_impact
-          doc.status              'ir'
           doc.reason1             reason_code.identifier
           doc.reason2             supplementary_reason_code.try(:identifier)
-        end
-        doc.drivers_report do
-          doc.citation         report.summons_or_warning_issued?
-          doc.weather          report.weather_conditions
-          doc.surr_cond        report.road_conditions
-          doc.lighting         report.light_conditions
-          doc.speed            report.speed
-          doc.point_of_contact report.damage_to_bus_point_of_impact
-          doc.total_pass       report.passengers_onboard
-          doc.ambulance        report.injured_passengers.any?(&:transported_to_hospital?)
-          doc.ov_tow           report.other_vehicle_towed_from_scene?
-          doc.pvta_tow         report.towed_from_scene?
+          doc.weather             report.weather_conditions
+          doc.surr_cond           report.road_conditions
+          doc.lighting            report.light_conditions
+          doc.direction           report.direction
+          doc.curb_distance       supervisor_incident_report.try(:curb_distance)
+          doc.vehicle_distance    report.vehicle_distance
+          doc.total_pass          report.passengers_onboard
+          doc.preventable         preventable
+          doc.police_on_scene     report.police_on_scene
+          doc.officer_info_taken  report.police_badge_number.present?
+          doc.citation            report.summons_or_warning_issued?
+          doc.wheelchair_involved report.wheelchair_involved
+          doc.ambulance           report.injured_passengers.any?(&:transported_to_hospital?)
+          doc.pvta_tow            report.towed_from_scene?
+          doc.ov_tow              report.other_vehicle_towed_from_scene?
+          doc.fatality            supervisor_report.try(:test_due_to_fatality)
+          doc.other_vehicle_info_taken   report.motor_vehicle_collision
```

Comment:
So if the incident was a motor vehicle collision that directly means that the other vehicle's info was taken? Just double-checking because it's a different pattern than most of the other "info taken" fields.

Reply:
Good catch. I think that using the field other_driver_license_number is probably the best. I think that the field motor_vehicle_collision was used before because all info of the other vehicle would be dependent upon the motor vehicle collision checkbox.

Reply:
Is other_vehicle_info_taken just supposed to be a boolean indicating whether or not the other vehicle's information was taken? That's how I read it, just making sure my interpretation is correct.

Reply:
Yes thats correct.